File: 程式語言 V2答案檔/C#/203.cs
using System;
class Class1 {
    static void Main() {
      	int input = Convert.ToInt32(Console.ReadLine());
        switch (input){
            case 1:
                Console.WriteLine("one");
                break;
            case 2:
                Console.WriteLine("two");
                break;
            case 3:
                Console.WriteLine("three");
                break;
            case 4:
                Console.WriteLine("four");
                break;
            default:
                Console.WriteLine("error");
                break;
        }
    }
}

File: 程式語言 V2答案檔/Go/203.go
package main
import "fmt"

func main(){
	var index int
	fmt.Scanln(&index)
	if index <= 0 || index > 4{
		fmt.Println("error")
		return
	}
	a := [4]string{"one", "two", "three", "four"}
	fmt.Println(a[index-1])
}